server/index.js

```diff
@@ -1,4 +1,6 @@
-require('dotenv').config({ path: '.env.development' });
+const envFile = `.env.${process.env.NODE_ENV || 'development'}`
```

Comment:
Do we event want to support env files for multiple environments? I guess `.env` should be enough.